autogen/docs/behaviorspace.md.mustache

```diff
@@ -9,6 +9,23 @@ This guide has three parts:
 - [**Advanced Usage**](#advanced-usage): How to use BehaviorSpace from the command
   line, or from your own Java code.
 
+A number of new features were introduce in NetLogo 6.4.0:
+
+- [**Subexperiment syntax**](#subexperiment-syntax): A syntax for allowing parameter
+ combinations to be run separately, rather than being expanded combinatorically.
+- [**Run metrics when**](#run-metrics-when): The recording of measurements can now 
+be controlled by a reporter.
+- [**Importing and exporting**](#importing-and-exporting): Experiments can now be exported to 
+an XML file that can be used when running headlessly. Experiments can also be imported into a
+ model.
+- [**Statistics output**](#statistics-output): The mean and standard deviation
+of data from repetitions can be saved in an output file.
+- [**Lists output**](#lists-output): List data can be output in a file with one list
+ element per cell.
+- [**Paused experiments**](#paused-experiments): Experiments can now be paused and resumed.
+
+Additional minor changes can be found by searching this page for *(Since 6.4)* 
```

Comment:
6.4.0 rather than 6.4 just for consistency, maybe?

Reply:
Change first instance to 6.4. The NetLogo dictionary contains only 2 levels for its "since" references